File: FrontEnd/src/app/features/cashiers/transactions/classes/dtos/cashier-write-dto.ts
import { Guid } from 'guid-typescript'

export interface CashierWriteDto {

    cashierId: Guid
    date: string
    companyId: number
    safeId: number
    entry: string
    amount: number
    remarks: string
    isDeleted: boolean
    putAt: string

}

File: API/Features/Cashiers/Transactions/Dtos/CashierWriteDto.cs
using System;
using API.Infrastructure.Interfaces;

namespace API.Features.Cashiers.Transactions {

    public class CashierWriteDto : IMetadata {

        public Guid CashierId { get; set; }
        public int CompanyId { get; set; }
        public int SafeId { get; set; }
        public string Date { get; set; }
        public string Entry { get; set; }
        public decimal Amount { get; set; }
        public bool HasDocument { get; set; }
        public string Remarks { get; set; }
        public bool IsDeleted { get; set; }
        public string PostAt { get; set; }
        public string PostUser { get; set; }
        public string PutAt { get; set; }
        public string PutUser { get; set; }

    }

}